notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>missing the f"...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/477/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>